src/components/Sum.jsx

```diff
@@ -1 +1,18 @@
-// export function Sum() {}
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a = 0,
+  b = 0,
```

Comment:
```suggestion
  a,
  b,
```
unnecessary if you set defaultProps